packages/components/pie-button/src/button.scss

```diff
@@ -189,20 +193,24 @@
     }
 
     &[variant='outline'] {
-        --btn-bg-color: var(--dt-color-container-default);
+        --btn-bg-color: transparent;
```

Comment:
I think the outline version should have a background color instead of being transparent. Otherwise, it might not be visible if the background isn't white. Also, the outline-inverse variant is meant to be transparent, so that should be used for non-white backgrounds as far as I know

<img width="176" alt="Screenshot 2023-09-04 at 11 01 51" src="https://github.com/justeattakeaway/pie/assets/28605803/1cc1d523-7f13-4a49-8bd3-780083616091">

snacks example:

<img width="763" alt="Screenshot 2023-09-04 at 11 04 57" src="https://github.com/justeattakeaway/pie/assets/28605803/b3c47930-61d1-420e-8635-d8f036efd97e">

Reply:
Never mind, I checked with @RLumleyJET and it should have a transparent background as the outline button will only used on 'light' colours and the inverse-outline will be used on the 'darker' colours"